### ISSUE TITLE
Rules regexes as separate json file

### ISSUE THREAD

**Issue by arroo** [tbusser/jscc#10]
Would it be possible to get the rules regexes made and used from a separate json file?

**Comment by tbusser** [comment]
Yeah, that is definitely on my to-do list. It will require a bit of work as the DataStore now has these regexs as RegExp objects. They'll have to converted to strings and used to create the RegExp objects once they're loaded. 

If you can convert the current RexExp objects to strings and initialize the RegExp object in the DataStore init method it would save me a bundle of time and I would be able get your request done quickly.
